code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -278,19 +276,7 @@ GLOBAL_LIST_EMPTY(ts_spiderling_list)
 	name += " ([rand(1, 1000)])"
 	real_name = name
 	msg_terrorspiders("[src] has grown in [get_area(src)].")
-	if(is_away_level(z))
-		spider_awaymission = TRUE
-		GLOB.ts_count_alive_awaymission++
-		if(spider_tier >= 3)
-			ai_ventcrawls = FALSE // means that pre-spawned bosses on away maps won't ventcrawl. Necessary to keep prince/mother in one place.
-		if(istype(get_area(src), /area/awaymission/UO71)) // if we are playing the away mission with our special spiders...
-			spider_uo71 = TRUE
-			if(world.time < 600)
-				// these are static spiders, specifically for the UO71 away mission, make them stay in place
-				ai_ventcrawls = FALSE
-				spider_placed = TRUE
-	else
-		GLOB.ts_count_alive_station++
```

Comment:
This is what I mean by stuff that needs a refactor. There shouldnt be a ton of area checks and stuff to determine if its AI, a `/ai` subtype is really needed.

Also `world.time` checks to see if something was manually placed, instead of using `mapload`.